third_party/move/move-compiler-v2/tests/bytecode-generator/v1-commands/pop_weird_2.move

```diff
@@ -0,0 +1,15 @@
+module 0x8675309::A {
```

Comment:
This test is derived from pop_weird to reveal the complete error message from V1.

Reply:
As the error message shows this is actually a bug, not an error message. Can we check to fix it?

Reply:
Sure, filed an issue for it: https://github.com/aptos-labs/aptos-core/issues/12099

Reply:
A fix is added to this PR: https://github.com/aptos-labs/aptos-core/pull/12085/commits/aecf7ad118c22b4ee56cdd151522bb67a253fa0f